api/src/main/java/org/ccci/idm/user/DefaultUserManager.java

```diff
@@ -148,7 +148,7 @@ public void deactivateUser(final User user) throws UserException {
         user.removeFacebookId(original.getFacebookId());
 
         // update the user object
-        this.userDao.update(original, user);
+        this.userDao.update(original, user, User.Attr.EMAIL, User.Attr.FLAGS, User.Attr.FACEBOOK);
     }
```

Comment:
In fact, does this not fix a bug since the current version uses the `MASK_DEFAULT = ImmutableSet.<String>builder().addAll(MASK.get(Attr.EMAIL)).addAll(MASK.get
            (Attr.NAME)).addAll(MASK.get(Attr.FLAGS)).build();` which would not have handled updating the removed Facebook id?